xbrowse_server/base/model_utils.py

```diff
@@ -130,13 +131,13 @@ def _deprecated_get_or_create_saved_variant(xpos=None, ref=None, alt=None, famil
 
 XBROWSE_TO_SEQR_ADDITIONAL_ENTITIES_MAPPING = {
     "VariantTag": {
-        "saved_variant": _deprecated_get_or_create_saved_variant
+        "saved_variants": _deprecated_get_or_create_saved_variant
```

Comment:
make sure to change the return type of `_deprecated_get_or_create_saved_variant` from a single variant to a list